index.bs

```diff
@@ -18,18 +18,53 @@ Version History: https://github.com/w3c/orientation-sensor/commits/gh-pages/inde
 Indent: 2
```

Comment:
the perspective of the phone sides on that picture in comparison with the earth, looks all wrong. The phone looks like it lays flat on the surface, but the sides tell another story... the sides should be parallel with the z axis